bin/edit_user_role.ts

```diff
@@ -17,9 +17,11 @@ import { getConfig } from '../config';
 
 process.on('unhandledRejection', (err) => { throw err; });
 
-const { userId, oid, add, rm } = parse(process.argv.slice(2));
+const { _: args } = parse(process.argv.slice(2));
+const [userId, orgId, add, rm] = args.map((x) => x.split('=')[1]);
```

Comment:
Ah, I see that I've missed something from the docs.

Usage is actually as follows:
```
$ npm run exec FILEPATH -- [--userId=a] [--orgId=b] ...
```
Note the `--`, which causes `npm` to pass the arguments following it to the process it executes rather than trying to consume them itself.

Including the `--` means that the arguments are correctly parsed. I can update the comment block at the top of the file to reflect this (and also at the top of the other scripts).

Reply:
Makes sense. The additional `--` does make this script setup more tedious. IMO it will be simpler to add values directly into scripts.